### PR TITLE
show validation error if the columns to export are greater than 255

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/flows/views.py [hunk 1]
```diff
@@ -918,6 +919,19 @@ def clean(self):
...
+                    columns = []
+                    flows = cleaned_data['flows']
+
+                    with SegmentProfiler("get columns"):
+                        for flow in flows:
+                            columns += flow.get_columns()
+
+                    # we limit to 75 since each takes 3 columns and we reserve 20 columns for other fields
+                    if len(columns) > 75:
+                        raise forms.ValidationError(_("You selected many flows, their results columns "
```
Comment (ycleptkellan) [inline temba/flows/views.py]: Sweet. Do we want to prompt the user to deselect flows, e.g. "The selected flows' results exceed the 255-column limit. Please deselect one or more flows"?

Comment (norkans7) [inline temba/flows/views.py]: Okay

File: temba/flows/tests.py [hunk 2]
```diff
@@ -739,6 +739,14 @@ def test_export_results(self):
...
                                             c1_run1_first, c1_run1_last, "Orange", "orange", "orange"], tz)
 
+        # validate we have not more than the maximum columns possible
+        with patch('temba.flows.models.Flow.get_columns') as mock_get_columns:
+            mock_get_columns.return_value = ["column %s" % i for i in range(100)]
+
+            response = self.client.post(reverse('flows.flow_export_results'), dict(flows=[self.flow.pk]), follow=True)
+            self.assertFormError(response, 'form', None, "The selected flows' results exceed the 255-column limit. "
+                                                         "Please deselect one or more flows")
+
```
Comment (ericnewcomer) [inline temba/flows/tests.py]: How about:
"This export exceeds the maximum number of columns (255). Please remove one or more of the flows from the export to continue."

**Comment by ericnewcomer** [comment]
Nice, slayin' those sentry dragons 🐲

**Pull request merged**